hledger-lib/hledger_csv.m4.md

```diff
@@ -279,8 +279,8 @@ to the amount
 (TODO: <s>when there is an amount</s>).
 Or, you can use an `amountN` [field assignment](#field-assignment) for more control, eg:
 ```
-fields date,description,currency,amount_
-amount1 %amount_ %currency
+fields date,description,currency,amount1
+amount1 %amount1 %currency
```

Comment:
Seems ok, but now we are showing them that in a field assignment you can reference the same field name previously assigned by a field list, and add to its value. This would lead me (did lead me) to think that I can do the same thing with two field assignments, which is not the case. This is the "special case" I think we need to mention somehow.

Reply:
But actually this whole manual is in need of a cleanup, so let's not worry about it right now.